doc/source/security/encrypt-traffic.rst

```diff
@@ -362,92 +302,202 @@ If you do everything correctly,
 ``mysqldump`` will connect to DONOR with the SST user,
 generate a dump file, and import it to JOINER node.
 
-xtrabackup
-----------
+.. _encrypt-replication:
 
-This is the default SST method
-(the :variable:`wsrep_sst_method` is set to ``xtrabackup-v2``),
-which uses |PXB|_ to perform non-blocking transfer of files.
-For more information, see :ref:`xtrabackup_sst`.
+Encrypting Replication/IST Traffic
+-----------------------------------
 
-Encryption mode for this method is selected using the :option:`encrypt` option:
+Replication traffic refers to the following:
 
-* ``encrypt=0`` is the default value, meaning that encryption is disabled.
+* Write-set replication which is the main workload of |PXC|
+  (replicating transactions that execute on one node to all other nodes).
+* Incremental State Transfer (:term:`IST`) which
+  is copying only missing transactions from DONOR to JOINER node.
+* Service messages which ensure that all nodes are synchronized.
 
-* ``encrypt=1``, ``encrypt=2``, and ``encrypt=3`` have been deprecated.
+All this traffic is transferred via the same underlying communication channel
+(``gcomm``). Securing this channel will ensure that :term:`IST` traffic,
+write-set replication, and service messages are encrypted.
+(For IST, a separate channel is to be configured using the same configuration
```

Comment:
s/For IST, a separate channel is to be configured using the same configuration/For IST, a separate channel is configured using the same configuration/g